packages/esm-patient-vitals-app/src/vitals-biometrics-form/delete-vitals-biometrics-modal.component.tsx

```diff
@@ -0,0 +1,64 @@
+import { ModalBody, ModalHeader, ModalFooter } from '@carbon/react';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import React from 'react';
+import { Button } from '@carbon/react';
+import { deleteVitalsAndBiometrics, invalidateCachedVitalsAndBiometrics } from '../common/data.resource';
+import { createErrorHandler, showSnackbar } from '@openmrs/esm-framework';
+
+interface DeleteVitalModalProps {
+  closeDeleteModal: () => void;
+  patientUuid: string;
+  encounterUuid: string;
+}
+
+const DeleteVitalsBiometricsModal = ({ closeDeleteModal, encounterUuid }: DeleteVitalModalProps) => {
+  const { t } = useTranslation();
+  const [isDeleting, setIsDeleting] = useState(false);
+  const deleteVitalsBiometricsDialog = async () => {
+    const abortController = new AbortController();
+    try {
+      const response = await deleteVitalsAndBiometrics(encounterUuid, abortController);
+      invalidateCachedVitalsAndBiometrics();
+      showSnackbar({
+        isLowContrast: true,
+        kind: 'success',
+        title: t('vitalsAndBiometricsDeleted', 'Vitals and Biometrics Deleted'),
+      });
+    } catch (error) {
+      createErrorHandler();
```

Comment:
Huh we should really document [that function](https://github.com/openmrs/openmrs-esm-core/blame/c10b70bf30e3ecbb240ac6abc81b1a61caa99d55/packages/framework/esm-error-handling/src/index.ts#L30). It definitely does nothing as used here. As the name would suggest, the thing that it does is to create an error handler, which it returns. `reportError` would be appropriate, and then you could get rid of the `showSnackbar` here, because I think `reportError` handles the notification.